mdm/common/src/main/java/com/gentics/mesh/core/data/dao/PersistingContentDao.java

```diff
@@ -0,0 +1,50 @@
+package com.gentics.mesh.core.data.dao;
+
+import com.gentics.mesh.context.BulkActionContext;
+import com.gentics.mesh.context.InternalActionContext;
+import com.gentics.mesh.core.data.HibNodeFieldContainer;
+import com.gentics.mesh.core.data.branch.HibBranch;
+import com.gentics.mesh.core.data.node.HibNode;
+import com.gentics.mesh.core.data.user.HibUser;
+import com.gentics.mesh.core.db.Tx;
+import com.gentics.mesh.core.rest.common.ContainerType;
+import com.gentics.mesh.core.result.Result;
+
+public interface PersistingContentDao extends ContentDao {
```

Comment:
As a next step we could remove call to these methods and call the node dao directly

Reply:
Perhaps they have to stay in ContentDao...

Reply:
Done.